### DISCUSSION TITLE
우측 상태바가 나오는경우, 안나오는경우가 따로 존재하나요?

### DISCUSSION THREAD

**Discussion by jjongguet** [pronist/hELLO#107] [Q&A] [answered]
안녕하세요. <h1~h3> 태그 모아보기 기능 관련 질문입니다.
티스토리에서 직접 작성한 게시글은 모아보기가 작동하는 반면
notion에서 작성한 문서를, HTML문서로 변환하여 티스토리에 작성한 경우는 모아보기가 작동하지않았습니다. 
혹시 HTML문서로 변환하여 올리는경우는 모아보기기능을 제공하지않나요?

<img width="1693" alt="image" src="https://user-images.githubusercontent.com/48749100/173971090-ce54b49b-f140-4851-b049-159dfdc3ab79.png">
<img width="1693" alt="image" src="https://user-images.githubusercontent.com/48749100/173971112-cca80ec7-61dc-47e1-8073-ac89a68726e7.png">

**Comment by pronist** [comment, marked as answer]
@jjongguet4u 안녕하세요. 우선 목차의 경우 티스토리 에디터로 작성한 것에만 적용되도록 처리되어 있습니다. 다른 추가적인 요소들도 티스토리 기본 에디터를 기준으로 동작하도록 처리되어 있습니다. 아무래도 **타 플랫폼에 있는 글을 복사 & 붙여넣기 하는 경우와 티스토리 에디터로 작성하는 경우는 HTML 구조가 전반적으로 다릅니다.** 따라서 지금 현 시점에서는 해당 스킨에서는 티스토리 에디터로 작성한 글에만 대응하도록 되어 있습니다. 

추후 일반화를 위해 패치를 할 수는 있지만 스킨 사용자가 어떤 플랫폼에서 글을 복사할 지는 아무래도 예상하기 어렵기 때문에 대응하지 못하는 경우가 많습니다. 때로는 임의로 HTML 을 추가할 수도 있고 말이죠. 넓은 범위를 일반화했다가 문제가 되었던 점은 사용자가 컨텐츠를 Gist 와 같이 스크립트를 통해 포함하는 경우, 컨텐츠 고유 스타일까지 변경되어서 디스플레이가 정상적으로 되지 않는 사례도 있었습니다. 따라서 어느 정도는 타 플랫폼에서 글을 붙여넣는 경우에는 제약이 있음을 알려드립니다.
